chart/princeton-manifold/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
+version: 0.1.0
+type: application
+name: princeton-manifold
+description: An implementation of the Manifold application for Princeton University
+appVersion: "v8.1.1"
+dependencies:
+- name: postgresql
```

Comment:
Note: We will likely not need the postgres chart to deploy, we will deploy with an external database configuration, ex: https://github.com/scientist-softserv/hykuup_knapsack/blob/main/ops/production-deploy.tmpl.yaml#L266